include/roaring/portability.h

```diff
@@ -27,11 +27,10 @@
 #include <malloc.h> // this should never be needed but there are some reports that it is needed.
 #endif
 
-#if __SIZEOF_LONG_LONG__ != 8
+#if defined(__SIZEOF_LONG_LONG__) && __SIZEOF_LONG_LONG__ != 8
```

Comment:
Yes, in fact CMake will select 32 for a target by default. Caught me off-guard for this project. AFAIK there is no reliable way to force CMake to target 64 on Windows.

In 32-bit, compilation will fail while looking for built-ins. We could check for the existence of _WIN32 and the absence of _WIN64 if we wanted to print something friendlier.